# ไม่มี input
---
bool success
string message
